reference/book.toml

```diff
@@ -3,3 +3,7 @@ authors = ["The Rust Project Developers"]
 multilingual = false
 src = "src"
 title = "Unsafe Code Guidelines Reference"
+
+[output.html]
+git-repository-url = "https://github.com/rust-lang/unsafe-code-guidelines"
+git-repository-icon = "fa-github"
```

Comment:
What does this "icon" thing do?
https://github.com/rust-lang/reference/blob/master/book.toml does not seem to have it.

Reply:
It configures the icon, but I just saw that it defaults to `fa-github` (https://rust-lang.github.io/mdBook/format/configuration/renderers.html#html-renderer-options).
I'll remove it.